model/src/main/java/org/cbioportal/model/AlterationEnrichment.java

```diff
@@ -12,14 +12,14 @@
     private String hugoGeneSymbol;
     private String cytoband;
     @NotNull
-    private Integer alteredCount;
-    @NotNull
-    private Integer unalteredCount;
-    @NotNull
     private String logRatio;
     @NotNull
     private BigDecimal pValue;
-
+    @NotNull
+    private AlterationCount set1AlterationCount;
```

Comment:
sorry to be pedantic but I think it does matter - its really `set1AlteredCount`, because its not the number of alterations in set1, right?

Reply:
seems like `AlterationCount` now wraps both `alteredCount` and `profiledCount`. So, it is not only the `alteredCount` anymore, but I don't know what's the best name for it.

Reply:
I see. hm. countInfo? countSummary? Something like that? I do think it's misleading to use the word "alteration"

Reply:
Looking at this again, I also agree that using the word `alteration` is confusing here. @angelicaochoa do you have suggestions for renaming?

Reply:
@averyniceday hmm perhaps `CountSummary` as @adamabeshouse suggested. Seems general but to the point